### PR TITLE
feat(ant-design-vue): support `css-in-js` import style

### PR DESCRIPTION
**Pull request by JohnsonHuang4396**
### Description

Support Ant-Design Vue 4.x

Previously, AntDesignVueResolver could only import CSS files, but now supports the introduction of CSS inJS files

### CODE REVIEW

**Comment by JohnsonHuang4396** [comment]
Found this is no need ToT

**Comment by ycs77** [comment]
I upgraded to ant-design-vue v4 and had an error:

```
[vite]: Rollup failed to resolve import "ant-design-vue/es/config-provider/style/css" from "D:/src/App.vue?vue&type=
script&setup=true&lang.ts".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
```

Now I using the `less` import style to fix it temporarily:

```
AntDesignVueResolver({
  importStyle: 'less',
}),
```

I think this PR could fix it, @JohnsonHuang4396 could you reopen it?

**Comment by ycs77** [comment]
And now the ant-design-vue is released v4, could you set the `importStyle` default to `CSSinJS`?

**Comment by JohnsonHuang4396** [comment]
I'm glad that this PR is helpful to you, but I think the CSSinJS option is exactly the same as the Less option, just with different name. But I can't figure out a better way, so I closed this PR.
Maybe you could give me some clue?🧐

**Comment by ycs77** [comment]
Thanks for the response. I think that although the functionality is the same, the option value still has meaning to the user. Just like for package users "less" and "CSSinJS" are different. 🤔

File: src/core/resolvers/antdv.ts [hunk 1]
```diff
@@ -182,7 +182,7 @@ export interface AntDesignVueResolverOptions {
    *
    * @default 'css'
    */
-  importStyle?: boolean | 'css' | 'less'
+  importStyle?: boolean | 'css' | 'less' | 'CSSinJS'
```
Comment (antfu) [inline src/core/resolvers/antdv.ts:185]:
```suggestion
  importStyle?: boolean | 'css' | 'less' | 'css-in-js'
```

**Pull request merged**